hack/run-all.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Run all the important scripts in a sensible order,
+# handy for switching branches, etc
+wd="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
So the `$( )` is a sub-shell and you can put additional unescaped quotes in it.
My bash lesson for the day. 😄 
Neat it works, not sure if its a good practice.  Will def update to look like our other scripts.